sdk/botservice/Microsoft.Azure.Management.BotService/src/Generated/Models/MsTeamsChannelProperties.cs

```diff
@@ -57,35 +48,17 @@ public MsTeamsChannelProperties(bool isEnabled, bool? enableMessaging = default(
         /// </summary>
         partial void CustomInit();
 
-        /// <summary>
-        /// Gets or sets enable messaging for Microsoft Teams channel
-        /// </summary>
-        [JsonProperty(PropertyName = "enableMessaging")]
-        public bool? EnableMessaging { get; set; }
-
-        /// <summary>
-        /// Gets or sets enable media cards for Microsoft Teams channel
-        /// </summary>
-        [JsonProperty(PropertyName = "enableMediaCards")]
-        public bool? EnableMediaCards { get; set; }
-
-        /// <summary>
-        /// Gets or sets enable video for Microsoft Teams channel
-        /// </summary>
-        [JsonProperty(PropertyName = "enableVideo")]
-        public bool? EnableVideo { get; set; }
-
         /// <summary>
         /// Gets or sets enable calling for Microsoft Teams channel
         /// </summary>
         [JsonProperty(PropertyName = "enableCalling")]
         public bool? EnableCalling { get; set; }
 
         /// <summary>
-        /// Gets or sets enable messaging for Microsoft Teams channel
+        /// Gets or sets webhook for Microsoft Teams channel calls
         /// </summary>
-        [JsonProperty(PropertyName = "callMode")]
-        public string CallMode { get; set; }
+        [JsonProperty(PropertyName = "callingWebHook")]
+        public string CallingWebHook { get; set; }
```

Comment:
This will result in a breaking change do you have the link to the spec change associated with this?

Reply:
Azure/azure-rest-api-specs#3478 this has been changed for over 2 years